app/src/main/java/io/musicorum/mobile/screens/Scrobbling.kt

```diff
@@ -57,7 +57,7 @@ fun Scrobbling(scrobblingViewModel: ScrobblingViewModel = hiltViewModel()) {
     val state = rememberLazyListState()
     val firstItemOffset = remember { derivedStateOf { state.firstVisibleItemScrollOffset } }
     val firstItemIndex = remember { derivedStateOf { state.firstVisibleItemIndex } }
-    val interpolated = Utils.interpolateValues(firstItemOffset.value.toFloat(), 0f, 100f, 122f, 46f)
+    val interpolated = Utils.interpolateValues(firstItemOffset.value.toFloat(), 0f, 200f, 122f, 46f)
```

Comment:
Be sure that the offset value can go up to 200

Reply:
working fine so i guess it can't go wrong